hadoop-ozone/dist/src/main/smoketest/omha/om-roles.robot

```diff
@@ -53,3 +53,15 @@ List om roles as JSON without OM service ID passed
                                     Assert Leader Present in JSON   ${output_without_id_passed}
     ${output_without_id_passed} =   Execute And Ignore Error        ozone admin --set=ozone.om.service.ids=omservice,omservice2 om roles --json
                                     Should Contain                  ${output_without_id_passed}      no Ozone Manager service ID specified
+
+List om roles as TABLE with OM service ID passed
+    ${output_without_id_passed} =   Execute                         ozone admin om roles --table
```

Comment:
@slfan1989 ,  this should be "output_with_id_passed".  BTW, the first command "ozone admin om roles --table" doesn't have service ID set, why it's under this "with OM service ID passed" category?

Reply:
Thank you for helping review the code. The issue was due to my oversight, and I have updated the unit tests.

Reply:
@ChenSammi Can you help review this pr again? thank you very much!